Dockerfile.origin

```diff
@@ -0,0 +1,9 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
```

Comment:
The upsteam project is built using Go 1.14, but container image `registry.svc.ci.openshift.org/openshift/release:golang-1.14` does not exist yet.

I could ...
1. Not worry about it right now
2. Figure out how to get the Go 1.14 container image create
3. Carry a patch to update go.mod and go.sum to use Go 1.13

I'm open to suggestions on the preferred approach.